psydac/api/tests/test_2d_biharmonic.py

```diff
@@ -74,9 +74,9 @@ def run_biharmonic_2d_dir(solution, f, dir_zero_boundary, ncells, degree, backen
 
     # Error norms
     error  = u - solution
-    l2norm = Norm(error, domain, kind='l2')
-    h1norm = Norm(error, domain, kind='h1')
-    h2norm = Norm(error, domain, kind='h2')
+    l2norm = SemiNorm(error, domain, kind='l2')
+    h1norm = SemiNorm(error, domain, kind='h1')
+    h2norm = SemiNorm(error, domain, kind='h2')
```

Comment:
I think it is confusing to write `SemiNorm(..., kind='l2')` in the code, given that it is equivalent to `Norm(..., kind='l2')`. In this case we should prefer the `Norm` class wherever possible.

@campospinto  @saidctb Do you agree?

Reply:
I have replaced L2 seminorm with L2 norm in 975404e

Reply:
okay, it is fine with me